agent/bench-scripts/tests/pbench-linpack/test-65.pre

```diff
@@ -3,6 +3,6 @@
 # Setup a fake install of linpack but with a stub for the expected executable.
 _linpack_dir=${_testtmp}/linpack
 mkdir ${_linpack_dir} || exit 1
-printf -- "#!/bin/bash\nexit 0\n" > ${_linpack_dir}/xlinpack_xeon64 || exit 1
+printf -- "#!/bin/bash\nfor ext in out meta finished; do touch '${_linpack_dir}/linpack'.$ext; done\nexit 0\n" > ${_linpack_dir}/xlinpack_xeon64 || exit 1
```

Comment:
In the normal execution of `xlinpack_xeon64`, does that program create the `linpack.out`, `.meta`, and `.finished`?  If not, what is the advantage of mocking them here?  Won't the `linpack.sh` take care of that?

Reply:
Hello Peter, it was just my attempt to create it here as I wasn't sure the mocked script is creating these. So this change should not be here (and it didn't help anyway)

Reply:
Now that this script-let actually does something, I suspect that we no longer want the `exit 0\n` in there.

Reply:
(Obviously, I with draw my comment if you're going to withdraw this change....  😁)